app/src/test/java/com/codesoom/assignment/application/AuthenticationServiceTest.java

```diff
@@ -0,0 +1,120 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.domain.User;
+import com.codesoom.assignment.domain.UserRepository;
+import com.codesoom.assignment.dto.UserLoginData;
+import com.codesoom.assignment.errors.PasswordNotCorrectException;
+import com.codesoom.assignment.errors.UserNotFoundException;
+import com.codesoom.assignment.utils.JwtUtil;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+class AuthenticationServiceTest {
+
+    private static final String SECRET = "12345678901234567890123456789012";
+    private static final String EXISTED_EMAIL = "example@login.com";
+    private static final String NOT_EXISTED_EMAIL = "not-existed@login.com";
+    private static final String CORRECT_PASSWORD = "password";
+    private static final String INCORRECT_PASSWORD = "incorrect";
+    private static final String ACCESS_TOKEN = "eyJhbGciOiJIUzI1NiJ9.eyJ1c2VyRW1haWwiOiJleGFtcGxlQ" +
+                                               "GxvZ2luLmNvbSJ9.VC6A_h3VehY_zvmxNGtgdwCc6zeFRBcdKSkj2rNB93Y";
+
+    private AuthenticationService authenticationService;
+
+    private UserRepository userRepository = mock(UserRepository.class);
+
+    private User user;
+
+    private UserLoginData userLoginData;
+
+    @BeforeEach
+    void setUp() {
+        JwtUtil jwtUtil = new JwtUtil(SECRET);
+        authenticationService = new AuthenticationService(jwtUtil, userRepository);
+
+        user = User.builder()
+                   .email(EXISTED_EMAIL)
+                   .password(CORRECT_PASSWORD)
+                   .build();
+    }
+
+    @Nested
+    @DisplayName("login 메소드는")
+    class Describe_login {
+
+        @Nested
+        @DisplayName("유효한 사용자 로그인 정보가 주어지면")
+        class Context_with_valid_userLoginData {
+
+            @BeforeEach
+            void prepareUserLoginData() {
+                userLoginData = UserLoginData.builder()
+                                             .email(EXISTED_EMAIL)
+                                             .password(CORRECT_PASSWORD)
+                                             .build();
+
+                given(userRepository.findByEmail(EXISTED_EMAIL)).willReturn(Optional.of(user));
+            }
+
+            @Test
+            @DisplayName("이메일로 사용자를 찾고 Access Token을 리턴한다")
```

Comment:
`이메일로 사용자를 찾고`는 `로그인 처리`로 추상화해서 표현할 수 있겠죠.
```suggestion
            @DisplayName("로그인 처리하고 Access Token을 리턴한다")
```